src/app/sale/models/sale.model.ts

```diff
@@ -0,0 +1,44 @@
+// Sales model
+class Sell {
```

Comment:
You should create an interface to implement in the Sell class.

Reply:
Thanks, I will! :)